kitodo-mediaserver-fileserver/src/main/java/org/kitodo/mediaserver/fileserver/controller/FileController.java

```diff
@@ -138,25 +139,34 @@ public void getFile(
 
                 // if work is disabled and request is not on METS/MODS, block the request
                 if (!isAllowedIpAddress(senderIp, subnets) && !StringUtils.endsWith(derivativePath, workId + ".xml")) {
-                    message = "Work with id " + workId + " is disabled or the source IP address is not allowed";
-                    LOGGER.info(message);
-                    throw new HttpForbiddenException(message);
+                    if ((new File(fileserverProperties.getDisabledWorkImagePath()).isFile())) {
+                        workDisallowed = true;
```

Comment:
I'm thinking about this. On a 404 you can set page content too. Can and should we set the status code to 403 AND set the image? Not sure if this would work with Kitodo.Presentation.